src/testIntegration/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/IntegrationTest.kt

```diff
@@ -107,9 +117,9 @@ abstract class IntegrationTest {
     contactRepository.deleteAll()
   }
 
-  @BeforeEach
-  fun `turn off random checks`() {
-    doReturn(false).whenever(randomCheckService).requiresRandomCheck()
+  @AfterEach
```

Comment:
We need to reset all of our mock and spy behaviour after each test as I found that we were not doing this and the behaviour of the repository spies in my new test (throwing exceptions) was bleeding into other tests and making them fail.